### PR TITLE
TIE-43: Trigger CD init with new field

### PR DESCRIPTION
**Pull request by zigomir**
Accommodates breaking change in https://github.com/sleuth-io/sleuth/pull/7283

### CODE REVIEW

File: internal/gqlclient/models.go [hunk 1]
```diff
@@ -189,8 +189,9 @@ type MutableCodeChangeSource struct {
 }
 
 type CreateCodeChangeSourceMutationInput struct {
-	ProjectSlug       string `json:"projectSlug"`
-	InitializeChanges bool   `json:"initializeChanges"`
+	ProjectSlug       		string `json:"projectSlug"`
+	InitializeChanges 		bool   `json:"initializeChanges"`
+	InitializeChangesNow 	bool   `json:"initializeChangesNow"`
```
Comment (JureMalovrh) [inline internal/gqlclient/models.go:194]: This is formated a bit weird, probably `make format` will get this in order

Comment (zigomir) [reply internal/gqlclient/models.go:194]: Uh, I only see it now in diff (tabs vs spaces 😊), will fix, tnx!

Comment (JureMalovrh) [reply internal/gqlclient/models.go]: `gofmt` to the rescue :D

**Review by JureMalovrh — APPROVED** [review]

**Comment by zigomir** [comment]
Closing because of https://github.com/sleuth-io/sleuth/pull/7283#pullrequestreview-1292702735

**Pull request closed without merging**